projects/utils/src/api/git/profile.ts

```diff
@@ -0,0 +1,8 @@
+import { IGitSystem } from './api'
+import { LibraryProfile } from '../../types'
+
+export const gitProfile: LibraryProfile<IGitSystem> = {
+  name: 'git',
+  methods: ['clone', 'checkout', 'init', 'add', 'commit', 'fetch', 'pull', 'push', 'reset', 'status', 'remote', 'log']
+  //events: ['currentFileChanged']
```

Comment:
As there is no events expected in a git plugin, you can remove the comment

Reply:
This has also been removed